package/network/services/igmpproxy/files/igmpproxy.init

```diff
@@ -79,18 +79,20 @@ igmp_add_firewall_routing() {
 }
 
 igmp_add_firewall_network() {
-	config_get network $1 network
 	config_get direction $1 direction
+	config_get zone $1 zone
+
+	[ ! -z "$zone" ] || return
 
 	json_add_object ""
 	json_add_string type rule
-	json_add_string src "$network"
+	json_add_string src "$zone"
```

Comment:
I think you need to add 'json_add_string family ipv4' in this section, otherwise ipv6 igmp rules get created.  Since igmp is ipv4 only (AFAIK) it makes little sense to add to the ipv6 firewall.

Reply:
The rules that were already in place were not created by me. However, I believe you are right in that igmp is ipv4 only, and hence I've added your suggestion to the init script to reflect this as well.